### ISSUE TITLE
Allow players to have multiple votes.

### ISSUE THREAD

**Issue by shacknetisp** [red-eclipse/base#403]
The discussion about this is here, it wasn't submitted as an issue yet: http://redeclipse.net/forum/viewtopic.php?f=4&t=68

**Comment by bonifarz** [comment]
Just to summarize some points from the forums:
* players could vote once for each suggested game (maybe show checkboxes)
* suggesting multiple games should be limited (maybe clear previous votes)
* some might still prefer to allow only one vote per player

**Comment by acz13** [comment]
:arrow_up: **Bump!**  
Honestly, if this doesn't take too much work, this should be good.  
This prevents a terrible mode & map (i.e. freestyle-instagib) from winning due to the other people being split on other ideas.

**Comment by restcoser** [comment]
agreed, this is a good idea (i think there should be a server var to switch the behaviour tho)

(just wanted to write tho, freestyle instagib is great :( leave it alone! :3)
